examples/cdn-example/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>LIFF Mock example</title>
+    <script
+      charset="utf-8"
+      src="https://static.line-scdn.net/liff/edge/versions/2.19.0/sdk.js"
```

Comment:
Why don't we use latest version?

Reply:
Currently liff-mock@1.0.2 supports LIFF SDK v2.19.0 or earlier.
https://github.com/line/liff-mock/blob/f7d6727338cbb7a866ec3c8d418819b665d4bd46/package.json#L34-L35

I'll add the support for the latest SDK in a later PR but this PR is enough to support 2.19.0.